app/components/LatestDataRefresh/LatestDataRefresh.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../styles/vars.scss";
+
+.sticky-container {
+  position: sticky;
+  width: 100%;
+  top: 0;
+  z-index: 1;
+
+  &:before {
+    content: "";
+    top: 0;
+    position: absolute;
+    height: 6.25rem;
+    width: 100%;
+    background: linear-gradient(180deg, #000000 0%, rgba(0, 0, 0, 0) 100%);
```

Comment:
Could this change to a white gradient on light mode please?
Might be needing another css var

Reply:
Yeah I am a little unsure on the light mode gradient, have a look and suggest ta